halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/config/v1/secrets/SecretSessionManager.java

```diff
@@ -69,6 +71,14 @@ public String decrypt(String filePathOrEncryptedString) {
     return secretManager.decrypt(filePathOrEncryptedString);
   }
 
+  public String validatingFileDecrypt(ConfigProblemSetBuilder ps, String filePath) {
+    String contents = secretManager.decrypt(filePath);
```

Comment:
This reads a little funny to me -- why are we checking to see if the output of decrypting the contents of a filepath matches the filepath?